src/Desktop.Analyzers/Desktop.Analyzers.md

```diff
@@ -48,6 +48,16 @@ Severity: Warning
 
 Help: [http://msdn.microsoft.com/library/ms182350.aspx](http://msdn.microsoft.com/library/ms182350.aspx)
```

Comment:
I have no objection on backtick-quoting all code in the description, but I think it'd be better if we do a complete scrub of all the analyzers.

What I meant is, this change is automatically generated by the a2md tool, all the strings are fetched from the analyzer DLLs directly. So if we want to do what you suggested, we need to update the string in the code.